dev/Breadcrumb/Breadcrumb.xaml

```diff
@@ -0,0 +1,115 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+
+    <!-- see https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font -->
+    <x:String x:Key="SplitButtonSecondaryButtonContent">&#xE76C;</x:String>
+    <x:String x:Key="SplitButtonSecondaryButtonDefaultContent">&#xE70D;</x:String>
+
+    <Style x:Key="BreadcrumbItemStyle" TargetType="local:BreadcrumbItem">
+
+        <Setter Property="UseSystemFocusVisuals" Value="True" />
+        <Setter Property="FocusVisualMargin" Value="-3" />
+        <Setter Property="IsTabStop" Value="True"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:BreadcrumbItem">
+
+                    <Button x:Name="PART_BreadcrumbItemButton">
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">
+                                <VisualState x:Name="Normal" />
+
+                                <VisualState x:Name="NormalRTL">
```

Comment:
These sound like it should not be in the common states visual group. Is there one more more groups that you can think of the following visual states falling in naturally ?